VSCode_Engine-Source_Windows/.vscode/c_cpp_properties.json

```diff
@@ -14,9 +14,9 @@
                 "SDL_WINDOWING"
             ],
             "windowsSdkVersion": "10.0.19041.0",
-            "compilerPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.29.30133\\bin\\Hostx64\\x64\\cl.exe",
+            "compilerPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.29.30133\\bin\\Hostx64\\x64\\cl.exe",
             "cStandard": "c11",
-            "cppStandard": "c++11",
+            "cppStandard": "c++14",
```

Comment:
What does this do? It doesn't move the project to 14. 
Is it mandatory for 2022?

Reply:
This is just to give Visual Studio Code a hint as to what's going on. I don't think it's mandatory. But it was set previously -- to a setting that is no longer correct -- so I figured I might as well update it.

Reply:
Note that this indicates the ISO C++ standard in use. In this case, c++14.